src/stories/components/custom-multi-select/custom-multi-select.tsx

```diff
@@ -125,7 +125,7 @@ export const CustomMultiSelect = ({
               placeholder="Search"
               value={searchText}
               onChange={(val) => setSearchText(val)}
-              handleCleanSearch={() => setSearchText('')}
+              handleClearSearch={() => setSearchText('')}
```

Comment:
What do you think about avoid inline function:
you can use handleClearSearch, callback and call that function
const  handleClearSearch=useCallback(()=>{
setSearchText('')
})
What do you think ????

Reply:
Let's do it